### ISSUE TITLE
[Findability] User does not have a path to the content or feature.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#75983]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** govcio-vfep
**Product:** GI Bill Comparison Tool
**Feature:** Specialized Mission Filter

## Findings details
**VA.gov Experience Standard - issue:** User does not have a path to the content or feature.
**VA.gov Experience Standard - category:** Findability
**Launch-blocking:** Yes
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When a user has set their browser zoom to 300% or higher, the "Filter your results" section is not visible on the screen. This excludes many low-vision users from using this feature.

Likewise, the feature is hidden on mobile devices. Many blind Veterans use a mobile device as their primary (and sometimes only) computing device. Hiding functionality on mobile devices is a poor UX practice that is likely to disproportionately exclude blind Veterans.

<details><summary>Link, screenshot or steps to recreate</summary>

Steps to reproduce:

1. Navigate to the [GI Bill Comparison Tool on staging](https://staging.va.gov/education/gi-bill-comparison-tool/).
2. Use ctrl + / cmd + to increase browser zoom to 300%. Note that the "Filter your results" section of content is no longer present.

</details>

## Recommended action
WCAG 1.4.10 requires support for all content up to 400% zoom, equivalent to a viewport 320 pixels wide.

## References

- **WCAG Success Criteria:** [WCAG 1.4.10 Reflow - Level AA](https://www.w3.org/TR/WCAG21/#reflow)
- **Modality:** Browser zoom
- **Design System Component:** N/A
- **Design System Pattern:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** Platform OCTO leadership asks that we document all issues we identify. If you believe that this issue is out of scope or not your team's responsibility, it is still your responsibility to resolve the issue. Please comment on Slack in your team channel tagging `@platform-governance-team-members`, providing an explanation and who you believe is responsible. Governance team will research and follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by arianaadili** [comment]
On 2/27, my team and I collaborated with the governance team to verify the modifications implemented in our code. Our front-end dev team configured the Zoom browser to 400% and higher to confirm the visibility of all filters and checkbox components for users. @briandeconinck granted approval for our changes. Enclosed are screenshots depicting our product in production when zoomed in at 400%.

<img width="1047" alt="Screenshot 2024-02-28 at 10 55 29 AM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/119622092/dc1db3b9-2707-4dfd-abc7-4303a4eb1b10">
<img width="1230" alt="Screenshot 2024-02-28 at 10 55 40 AM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/119622092/c546ed35-d679-4c77-be2f-e81425ea00ec">
<img width="1223" alt="Screenshot 2024-02-28 at 10 55 47 AM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/119622092/fa20f7d2-1582-46c8-b2c7-f5c5868894f5">
<img width="1197" alt="Screenshot 2024-02-28 at 10 56 14 AM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/119622092/3c986a37-b48b-4159-91e4-9e7a17d467f1">

**Comment by GcioNapoleon** [comment]
![Screenshot 2024-02-28 at 11 37 08 AM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/95312667/edabdde1-7c78-4182-8afb-b494d052d202)
